SimplePolicy/SimplePolicy/config.json

```diff
@@ -2,8 +2,10 @@
   "PluginConfiguration": {
     "MaxTransactionsPerBlock": 500,
     "MaxFreeTransactionsPerBlock": 20,
-    "MaxFreeTransactionSize": 1024,
     "FeePerExtraByte": 0.00001,
```

Comment:
It is a little bit strange to still keep `FeePerExtraByte` here.

However, the old `VerifySizeLimits1` limit was kept.